scripts/plot_pahfit

```diff
@@ -70,7 +70,7 @@ if __name__ == "__main__":
     obs_y = obs_y.value
 
     # read in the PAHFIT results
-    pmodel = PAHFITBase(obs_x, obs_y, filename=args.fitfilename)
+    pmodel = PAHFITBase(obs_x, obs_y, estimate_start=False, filename=args.fitfilename)
```

Comment:
Since `estimate_start=False` is the default, you do not need to put include this in the PAHFITBase call.

Reply:
Got it, thanks.